SVGKit-iOS.xcodeproj/project.pbxproj

```diff
@@ -2691,7 +2691,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/3rd-party-frameworks/**";
```

Comment:
This `FRAMEWORK_SEARCH_PATHS ` was kept in the project level. So any `$(inherited)` will inherite this value. However, we have 3 target (Static-iOS, Framework-iOS, Framework-tvOS), each target need different framework on target platform. So instead of keeping this value in the project level. We'd better move it into 3 target level.